pact/Pact/Core/IR/ModuleHashing.hs

```diff
@@ -224,7 +224,8 @@ encodeApp operator operands =
 
 encodeSchema :: Schema -> Builder
 encodeSchema (Schema n _sc) =
-  encodeQualName n
+  -- TODO: This changes the hash, is this fine @Jose?
```

Comment:
This function only  differs only on schemas with no `QualifiedName`, IIUC? (I'm not sure how often we encounter schemas with no `QualifiedName`, so I'm not sure if this change to the hash is ok).

Reply:
This entire function needs to go anyway. We need to restore source text hashing regardless.

Reply:
@imalsogreg this is correct. I do think this is the behaviour we want in this particular case OR fail in case we hit a case with no `ModuleName`.